.github/workflows/build-test.yaml

```diff
@@ -2612,6 +2612,8 @@ jobs:
         run: |
           set +e
 
+          touch not-an-app.airgap
```

Comment:
remove this? not sure why it was added. maybe for testing?

Reply:
this test will fail with an error about "not-an-app.airgap" missing, but the intention of the test is for that command to get far enough that the registry check fails, which is my understanding of the log grep right below it. I'm trying to provide the command with just enough to get that far